File: Day 22/Homework/Hm3.py
list1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
num1 = int(input("Enter number: "))
num2 = int(input("Enter number: "))

if num1 > num2:
    print(list1[num2:num1][::-1]) 
elif num2 > num1:
    print(list1[num1:num2])  
else:
    print([])  

File: Day 69/Homework/Hm3.js
let num1=Number(prompt("enter number: "));
let num2=Number(prompt("enter number: "));
let num3=Number(prompt("enter number: "));

if(num1 > num2 && num1 > num3){
    console.log("ყველაზე დიდი"+num1)
}
if(num2 > num3 && num2 > num1){
    console.log("ყველაზე დიდი"+num2)
}
if(num3 > num2 && num3 > num1){
    console.log("ყველაზე დიდი"+num3)
}